src/field.rs

```diff
@@ -76,26 +83,40 @@ pub trait FieldElement:
     /// Returns the prime modulus `p`.
     fn modulus() -> Self::Integer;
 
-    /// Writes the field element to the end of input buffer. Exactly `BYTES` bytes will be written.
+    /// Interprets the next `BYTES` little endian bytes from the input slice as an element of the
```

Comment:
cjpatton and I chatted about this and agreed to standardize on network order (big endian). However we do need an LE Field32 for compatibility with ENPA/Prio v2. We agreed to introduce a new field `LegacyField32` that is exactly the same as `Field32` except it uses a different encoding (I wound up using on `LittleEndianField32` to avoid the negative connotation of "legacy").